applications/FSIApplication/tests/convergence_accelerator_spring_MPI_test.py

```diff
@@ -142,7 +144,7 @@ def tearDown(self):
             gid_io.FinalizeResults()
 
         # clean temporary files
-        if mpi.rank == 0:
+        if KratosMultiphysics.DataCommunicator.GetDefault().Rank() == 0:
```

Comment:
if you use kratos utilities DeleteFileIfExisting then you don't need to check the rank

Reply:
Done. Thanks!.